src/Logic/TimeCode.cs

```diff
@@ -95,8 +93,7 @@ public int Minutes
             }
             set
             {
-                var ts = TimeSpan;
-                TimeSpan = new TimeSpan(0, ts.Hours, value, ts.Seconds, ts.Milliseconds);
+                _totalMilliseconds = TimeSpan.FromMinutes(value).TotalMilliseconds;

```

Comment:
I think the above code will remove hours, seconds and milliseconds... right?
(I've also added some unit tests regarding this)